engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/RuntimeManagementTest.scala

```diff
@@ -81,7 +81,11 @@ class RuntimeManagementTest extends InterpreterTest {
       totalOut = consumeOut
       while (totalOut.length < expect && round < 500) {
         round = round + 1
-        if (round % 10 == 0) forceGC();
+        if (round % 10 == 0) {
+          forceGC();
+        }
+        val res = eval("main a b = a * b").execute(7, 6)
```

Comment:
Looks like an accidental change. Otherwise, needs a comment explaining why this is being done here.

Reply:
FYI: https://github.com/enso-org/enso/pull/6335#discussion_r1169977256 - we need to execute some code to trigger the `TruffleSafepoint`.